README.md

```diff
@@ -40,4 +40,17 @@ We welcome contributions. Please see [`Building`](./documentation/Building.md) f
 development-build instructions. For large new
 features, please open an issue to discuss goals and approach in order
 to ensure a smooth PR integration and review process. All contributions
-must be licensed under the repository's [MIT License](./LICENSE).
\ No newline at end of file
+must be licensed under the repository's [MIT License](./LICENSE).
+
+# Testing
+
+* Unit tests: `pytest`
+* Demo notebooks:
+  * ```
+    cd apis/python
+    pip install -r test/ipynb/requirements.txt
+    pytest --nbmake test/ipynb
+    ```
+* Credentials:
+  * Some tests run on TileDB Cloud using your current environment variable `TILEDB_REST_TOKEN` -- you will need a valid API token for the tests to pass 
+  * For continuous integration, the token is configured for the `unittest` user and all tests should pass
```

Comment:
(Minor) I don't think we should add this back? I moved it to `Building.md` in this PR https://github.com/TileDB-Inc/TileDB-Vector-Search/pull/192/files - though if you want it in `README.md` instead I'm fine with that, but let's remove it `Building.md` then.

Reply:
Ah yea, since this PR got postponed I had to manually merge some changes

Reply:
No problem sounds good, thanks for updating